src/collections/Breadcrumb/BreadcrumbDivider.js

```diff
@@ -18,7 +18,10 @@ function BreadcrumbDivider(props) {
   const rest = getUnhandledProps(BreadcrumbDivider, props)
 
   if (icon) {
-    return <Icon className={classes} name={icon} {...rest} />
+    // TODO: After update <Icon> to API replace with this code:
+    // return <Icon className={classes} name={icon} {...rest} />
+
+    return <Icon className={[icon, classes].join(' ')} {...rest} />
```

Comment:
I've updated usage of `Icon`, but it's still failing `common.implementsIconProp`. It seems that this test it's not correct for such situation.